index.html

```diff
@@ -25,10 +25,11 @@ <h1>Luau Tickets</h1>
                     <input type="text" id="phonenumber" name="phonenumber" placeholder="Phone Number (xxx-xxx-xxxx)" autocomplete="off">
                     <input type="text" id="email" name="email" placeholder="Email" autocomplete="off">
                     <h3>Date of Birth</h3>
-                    <input type="date" id="birthday" name="birthday" placeholder="Date of Birth" autocomplete="off">
+                    <input type="date" id="birthday" maxlength="10" name="birthday" placeholder="Date of Birth (mm/dd/yyyy)" autocomplete="off">
```

Comment:
It may be redundant to add the text "date of birth" in the placeholder. I think just having "(mm/dd/yyyy)" is good enough!